lib/RequestSender.js

```diff
@@ -68,6 +68,11 @@ module.exports = class RequestSender
         this.clientTransaction = new Transactions.NonInviteClientTransaction(
           this._ua, this._ua.transport, this._request, eventHandlers);
     }
+    // Check if auth token is set in the configuration and no other Authorization header is set before
+    if (this._ua._configuration.auth && !this._request.headers.Authorization)
+    {
+      this._request.headers.Authorization = [ `Bearer ${this._ua._configuration.auth}` ];
+    }
```

Comment:
When using JWT authorization we are not acting upon server side 401/407 responses, so it is fine to just add the JWT if exists.

```suggestion
    // If authorization JWT is present, use it.
    if (this._ua._configuration.authorization_jwt)
    {
      this._request.headers.Authorization = [ `Bearer ${this._ua._configuration.auth}` ];
    }
```